service.games.discord.richpresence/resources/lib/discordIPC.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
the dummy `__init__.py` is no longer needed in python3, can you drop it from the submission?